RELEASE.md

```diff
@@ -0,0 +1,35 @@
+Release Instructions
+--------------------
+
+Pre-Commit
+----------
+1. pushd ngrok-java-native && fix-n-fmt && popd
+1. export NGROK_AUTHTOKEN="<your_authtoken>"; make all
```

Comment:
Thanks Ryan — can we get code snippets for parts of this document that users will need to copy / paste into their terminal?

Reply:
code snippets as in surround these with backticks? or something more specific?

Reply:
yes! surround with backticks. I had to reread this a few times to understand i should be pasting these in my terminal

Reply:
good call, added!